dynamic_rest/renderers.py

```diff
@@ -91,6 +91,9 @@ def process(result):
             c for c in context['columns'] if c not in self.COLUMN_BLACKLIST
         ]
         context['details'] = context['columns']
+        context['error_form'] = context['error_form']
+        if context['error_form']:
```

Comment:
Aah you're right. I mis-read the template logic, where you are already doing this check: `{% if error_form and not errors %}` corresponds to a single-upload error, and `{% if errors %}` correspond to bulk-upload error. :+1: